arangod/Cache/Manager.cpp

```diff
@@ -125,9 +125,11 @@ std::shared_ptr<Cache> Manager::createCache(CacheType type,
     uint64_t fixedSize = 0;
     switch (type) {
       case CacheType::Plain:
+        LOG_TOPIC(DEBUG, Logger::CACHE) << "Creating plain cache";
```

Comment:
Move this (and it's transactional counterpart) down to where the cache is created below, rather than here where the allocation size is determined.